testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -199,6 +202,17 @@ impl EventObserver {
                 None => json!(null),
             }
         };
+        (txid, success, raw_result, raw_tx, contract_interface_json)
```

Comment:
This is a pretty large tuple response (with several of the entries of the same type) -- I think it'd make sense here to add a new non-public struct like `ReceiptPayloadInfo` so that there isn't any confusion in the future if we need to add another entry here.